.github/workflows/android-build.yml

```diff
@@ -0,0 +1,47 @@
+name: Build Sample for Android
+
+on:
+  push
+
+jobs:
+  apk:
+    name: Apk Build
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          submodules: true
+
+      - name: Install dependencies
+        run: |
+              sudo apt update
+              sudo apt install openjdk-8-jdk ant android-sdk-platform-tools-common android-sdk ninja-build
+      - name: Setup Path
+        run: |
+              PATH="/usr/src/android-sdk/platform-tools:$PATH"
+              export ANDROID_HOME="/usr/src/android-sdk"
+      - name: Install and setup SDKManager
+        run: |
+              wget https://dl.google.com/android/repository/commandlinetools-linux-6609375_latest.zip
+              unzip commandlinetools-linux-6609375_latest.zip -d cmdline-tools
+              mkdir --parents "$ANDROID_HOME/cmdline-tools/latest"
+              sudo mv cmdline-tools/* "$ANDROID_HOME/cmdline-tools/latest/"
+              export PATH=$ANDROID_HOME/cmdline-tools/latest/bin:$PATH
+              sdkmanager --sdk_root=$ANDROID_HOME --install "platforms;android-31" "build-tools;31.0.0"
+              yes | sdkmanager --sdk_root=$ANDROID_HOME --licenses
+      - name: Update the build.gradle
+        run:  |
+               cp build.gradle SDL/android-project/app/
+
+      - name: Build APK
+        run: |
+              cd SDL/android-project/
+              ./gradlew assembleDebug
+
+      - name: Upload APK
+        uses: actions/upload-artifact@v3.1.2
+        with: 
+          name: artifact
+          path: ./sdl3-sample/SDL/android-project/app/build/outputs/apk/debug/app-debug.apk

```

Comment:
Is this path correct? On your workflow run (https://github.com/captain1947/sdl3-sample/actions/runs/5954064597) it says it can't find any artifacts. I think you need to remove `sdl3-sample` from the path.